server/src/main/java/org/elasticsearch/search/aggregations/bucket/composite/CompositeAggregator.java

```diff
@@ -260,7 +275,7 @@ public void collect(int doc, long zeroBucket) throws IOException {
     }
 
     private SingleDimensionValuesSource<?> createValuesSource(BigArrays bigArrays, IndexReader reader, Query query,
-                                                              CompositeValuesSourceConfig config, int sortRank, int size) {
+                                                              CompositeValuesSourceConfig config, int size, int sortRank) {
```

Comment:
was it a bug?

Reply:
not exactly, this was used to determine if we should use global ordinals or a binary value source for the leading composite source but the code has been refactored and global ordinals are now fast even when they are used as the leading source. I removed this code in https://github.com/elastic/elasticsearch/pull/36675/commits/b80398874b10bfce242373efe8b7f997ff610694 and we now use global ordinals all the time.